mdns.c

```diff
@@ -146,8 +146,6 @@ uint8_t *create_label(const char *txt) {
 
 	assert(txt != NULL);
 	len = strlen(txt);
-	if (len > 63)
```

Comment:
If you look at `rr_add_txt` in `mdns.c`, `create_label` is also used to fill in the child entries of a TXT record. Without this change, TXT entries longer than 63 chars are truncated to NULL and the server crashes when it attempts to encode such entries in a packet.

Maybe it is better to revert this and add a `create_txt_label` that does not have this check?